SonarQube.TeamBuild.PreProcessor/SupportedBootstrapperVersions.xml

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<BoostrapperSupportedVersions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://www.sonarsource.com/msbuild/integration/2015/1">
+  <SupportedVersion>1.0</SupportedVersion>
```

Comment:
"SupportedApiVersion" would be better - less chance of it being confused with the actual bootstrapper version.